examples/minesweeper/build.gradle.kts

```diff
@@ -0,0 +1,61 @@
+import org.jetbrains.kotlin.gradle.targets.js.nodejs.NodeJsRootExtension
+import org.jetbrains.compose.compose
+
+plugins {
+    kotlin("multiplatform") version "1.5.31"
+    id("org.jetbrains.compose") version "1.0.0-rc6"
```

Comment:
- use 1.0.0
- migrate to new template:
https://github.com/JetBrains/compose-jb/commit/dbd0a21949633302fcf98ad79c78896bbc6250ba#diff-7d46308093aec3efc8dd6079383a7b856efa79af87787a1fb2898504e0c4d027
![image](https://user-images.githubusercontent.com/5963351/145233475-73f13498-9083-4701-a61a-8e4d98d46a49.png)
-  add this example to https://github.com/JetBrains/compose-jb/blob/master/examples/validateExamples.sh